subiquity/ui/views/filesystem/guided.py

```diff
@@ -64,11 +64,11 @@ class LUKSOptionsForm(SubForm):
 
     def validate_password(self):
         if len(self.password.value) < 1:
-            return _("Password must be set")
+            return _("Passphrase must be set")
```

Comment:
That was mighty confusing in isolation but makes sense in the larger file.
A future refactor to s/password/passphrase would be welcome!

Reply:
Yes, the UI got me confused and by fixing it, I made the code confusing :upside_down_face:.
That was not so clever, I'll refactor in a follow on PR.